tokens/blau.json

```
@@ -668,6 +683,11 @@
       "type": "color",
       "description": "darkModeGrey"
     },
+    "backgroundContainerError": {
+      "value": "{palette.blauRed10}",
```

```suggestion
      "value": "{palette.darkModeGrey}",
```
Here the right color should be the same as `backgroundContainer` in dark mode, i.e. darkModeGrey

---

+1

* All `backgroundContainerError` tokens in light should have the same value that `errorLow`
* All `backgroundContainerError` tokens in dark should have the same value as `backgroundContainer`